resources/META-INF/plugin.xml

```diff
@@ -192,6 +192,11 @@
       <add-to-group group-id="RunMenu" anchor="after" relative-to-action="Stop"/>
     </group>
 
+    <!-- refactoring menu -->
+    <action class="io.flutter.actions.ExtractWidgetAction" id="Flutter.ExtractWidget" text="Flutter Widget...">
```

Comment:
Will this show up in an 'extract' menu? I'm wondering if the user will have enough context to know that `Flutter Widget...` means extract flutter widget. (if so, then lets keep the shorter text)

Reply:
The name and location is consistent with other `extract` refactorings.

![image](https://user-images.githubusercontent.com/384794/37919375-825741f6-30d8-11e8-8c6c-4c24b1f19c9b.png)